controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java

```diff
@@ -364,7 +364,7 @@ private ReaderGroupConfiguration getRGConfigurationFromRecord(final String scope
         return streamMetadataStore.getReaderGroupId(scope, rgName)
                 .thenCompose(rgId -> streamMetadataStore.getReaderGroupConfigRecord(scope, rgName, null, executor)
                         .thenApply(cfgRecord -> Controller.CreateReaderGroupResponse.newBuilder()
-                                .setConfig(encodeReaderGroupConfigRecord(scope, rgName, cfgRecord.getObject(), rgId))
+                                .setConfig(io.pravega.controller.server.rest.ModelHelper.encodeReaderGroupConfigRecord(scope, rgName, cfgRecord.getObject(), rgId))
```

Comment:
I know at some of the places we have brought in our wire objects over to this class, esp the response status codes. But all the encoding and decoding of complex objects should be handled at the model helper and our business logic should deal with the internal model classes exclusively and we should not have to encode it here. 

can we open a new github issue for this and i am ok with not addressing it here. 

thanks